app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx

```diff
@@ -0,0 +1,66 @@
+import { FormattedMessage as T } from "react-intl";
+import { InfoConfirmModal } from "modals";
+import { classNames } from "pi-ui";
+import styles from "./TrezorModals.module.css";
+
+const TrezorTogglePassphraseConfirmModal = ({
+  isGetStarted,
+  deviceLabel,
+  enablePassphraseProtection,
+  togglePassphraseConfirmCallback
+}) => {
+  const enable = enablePassphraseProtection ? "Enable" : "Disable";
+  const enabled = enablePassphraseProtection ? "enabled" : "disabled";
+  const ifEnabled = enablePassphraseProtection
+    ? " For your first sensitive operation, " +
+      "the device will ask for a passphrase. This pasphrase deterministically creates a " +
+      "new seed for your Trezor. If used with an already created wallet, subsequent " +
+      "actions will likely fail, as the device is now using a different account/wallet. " +
+      "As long as the device stays plugged in, further actions will reuse the first " +
+      "passed passphrase and not ask again."
+    : "";
```

Comment:
These need to be translatable.

Reply:
yeah, not even sure how to do that, will figure it out though

Reply:
Think this got it.